docker/templates/Dockerfile.test-database.m4

```diff
@@ -6,7 +6,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 # install_extensions.sh removes certain build dependencies that we need, so we
 # can't install everything here.
 # Note: curl is also a dependency of carton.
-RUN apt_install(`ca-certificates curl sudo')
+RUN apt_install(`ca-certificates curl sudo wget')
```

Comment:
`wget` should already be installed, but if it's not, is there a reason you can't use `curl`?